src/issue-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -21,10 +21,15 @@ const getTitle = (data: CreateIssueDetailsTextData): string => {
 };
 
 const getSelectorLastPart = (selector: string): string => {
-    let selectorLastPart = selector;
-    if (selector.lastIndexOf(' > ') > 0) {
-        selectorLastPart = selector.substr(selector.lastIndexOf(' > ') + 3);
+    const splitedSelector = selector.split(';');
+    let selectorLastPart = splitedSelector[splitedSelector.length - 1];
+
+    const childCombinator = ' > ';
+
+    if (selector.lastIndexOf(childCombinator) > 0) {
+        selectorLastPart = selector.substr(selector.lastIndexOf(childCombinator) + childCombinator.length);
```

Comment:
nit: instead of manipulating/changing an existing string; does it make sense to return the new string here directly and keep `selectorLastPart` as `const`?
Also does it make sense to store the condition in a variable that gives us some indication on what the condition is checking for? For example:
` const childCombinatorExists = selector.lastIndexOf(childCombinator) > 0`

Reply:
Changed from `let selectorLastPart` to `const selectorLastPart`.

As per the condition, it is a simple condition, as in, there is only one term, no AND nor OR operations. moving it to a const seems a little overkill.